Doc/using/windows.rst

```diff
@@ -25,7 +25,7 @@ As specified in :pep:`11`, a Python release only supports a Windows platform
 while Microsoft considers the platform under extended support. This means that
 Python |version| supports Windows 10 and newer. If you require Windows 7
 support, please install Python 3.8. If you require Windows 8.1 support,
-please install Python 3.12.
+please install Python 3.13.
```

Comment:
Responded to this on a different PR - according to [PEP 11](https://peps.python.org/pep-0011/), 3.13 doesn't support Windows 8.1, so we don't want to recommend it here. It works today (we believe, I don't think anyone is testing it), but we reserve the right to break it if necessary.